src/AlcoholTool.cpp

```diff
@@ -0,0 +1,123 @@
+/*
+ * AlcoholTool.cpp is part of Brewtarget and was written by Ryan Hoobler
```

Comment:
Unless the author _really_ wants to disown copyright, I agree it should be changed to the standard format:

    * AlcoholTool.cpp is part of Brewtarget, and is Copyright the following
    * authors 2009-2015
    * - Ryan Hoobler <rhoob@yahoo.com>